experiments/ppo_gridnet.py

```diff
@@ -340,8 +340,8 @@ def get_value(self, x):
 
     ## EVALUATION LOGIC:
     eval_queue = []
-    trueskill_df = pd.read_csv("league.csv")
-    trueskill_step_df = pd.read_csv("league.csv")
+    trueskill_df = pd.read_csv("po_league.csv")
+    trueskill_step_df = pd.read_csv("po_league.csv")
```

Comment:
We shouldn't have to worry about this anymore because the new script contains an output path for to CSVs: https://github.com/vwxyzjn/gym-microrts/blob/3d7a42f46efbd39a0b806388b8a445fbee48d00f/experiments/ppo_gridnet.py#L240.